packages/cli/src/run.test.ts

```diff
@@ -1,8 +1,10 @@
 import fixturez from "fixturez";
 import stripAnsi from "strip-ansi";
 import { exec } from "tinyexec";
+import { createRequire } from "node:module";
```

Comment:
We could replace this with https://www.npmjs.com/package/import-meta-resolve , but honestly using either is just equally fine given this is a test file anyway